src/test/java/com/xiaomi/infra/pegasus/client/TestBasic.java

```diff
@@ -2482,6 +2484,41 @@ public void delRange() throws PException {
     Assert.assertTrue(valueStr.contains("v_97"));
     Assert.assertTrue(valueStr.contains("v_98"));
     Assert.assertTrue(valueStr.contains("v_99"));
+
+    remainingValue.clear();
+    valueStr.clear();
+    DelRangeOptions delRangeOptions2 = new DelRangeOptions();
+
+    // test hashKey can't be null or ""
+    try {
+      client.delRange(
+          tableName, "delRange".getBytes(), "k1".getBytes(), "k2".getBytes(), delRangeOptions2);
+    } catch (Exception e) {
+      Assert.assertTrue(e.getMessage().contains("Invalid parameter: hash key can't be empty"));
```

Comment:
I can't see why `"delRange".getBytes()` is an empty hashkey... I think this test passed because it never catches any exception. The best practice to assert an exception is thrown is using `assertThrow`

```java
  Exception e = assertThrows(PException.class, () -> {
      client.delRange(
          tableName, "delRange".getBytes(), "k1".getBytes(), "k2".getBytes(), delRangeOptions2);
  });
  Assert.assertTrue(e.getMessage().contains("Invalid parameter: hash key can't be empty"));
```

https://howtodoinjava.com/junit5/expected-exception-example/